curp/src/server.rs

```diff
@@ -621,93 +655,79 @@ impl<C: 'static + Command, CE: 'static + CommandExecutor<C>> Protocol<C, CE> {
         &self,
```

Comment:
For the propose, we need another TODO, check the duplicate propose as one client may send the same request many times.